app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +38,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
   }
+
+  render() {
+    return (this.props.feedback.length) ? <FeedbackSummary feedback={this.props.feedback} /> : null;
```

Comment:
Ah, you're right. I mistakenly thought `this.props.feedback.length` would return `true` but it returns `0`. Thanks.